modular_splurt/code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -53,6 +53,87 @@
 	new_spawn.language_holder.understood_languages += /datum/language/draconic
 	new_spawn.language_holder.spoken_languages += /datum/language/draconic
 
+//Neutral slavers for hardcore doms players and their bitches
+/obj/effect/mob_spawn/human/lone_slaver
+	name = "Space Slaver"
+	roundstart = FALSE
+	death = FALSE
+	job_description = "NMC Agent"
+	icon = 'icons/obj/machines/sleeper.dmi'
+	icon_state = "sleeper_s"
+	short_desc = "You are a slaver of corporation which operating in 'the entertainment' industry."
+	flavour_text = "You have been sent to conduct various psychological experiments at a Night Moon Corporation outpost hidden from prying eyes.\
+	You did a great job in a NMC, or you just happened to be lucky, now you can consider that this is your vacation.\
+	Regardless of the task from the corporation, you can do whatever you want with your slaves. But you are still subordinate to your boss.\n\n"
+	important_info = "IMPORTANT!!! Any act with slaves is permitted, including forced intercourse, as long as they do not cross the line of hard-gross kinks \
+	as SCAT, GORE, BLOOD, DEATH, VORE. You need check OOC or ask in LOOC your slave. Your role involves close interactions with the slave. So whenever \
```

Comment:
This should be in line with general server rules for non-con, and not have it's own instructions here.
Players can do what ever they want with each other, with OOC consent.

Reply:
The purpose of this was to tell players that this is a ghostrole with close ERP interaction, and is not for an escape simulator. I'll rephrase the text.

Reply:
What happened? I only changed some text and now it's broken. And I can see some words from old text marked inside of your reviev here in the error log. I think because of it?

Reply:
> What happened? I only changed some text and now it's broken. And I can see some words from old text marked inside of your reviev here in the error log. I think because of it?

You'll probably have to put all of the text in one line or not put \ at the end, since \'s are used for specific characters on strings

Reply:
> > What happened? I only changed some text and now it's broken. And I can see some words from old text marked inside of your reviev here in the error log. I think because of it?
> 
> You'll probably have to put all of the text in one line 

Thank yoooou!!